### PR TITLE
[Docs site] Added a new value to search auto-focus

### PR DESCRIPTION
**Pull request by kodster28**
Should work for `CTRL` + `K` (in addition to the previous defaults of `SHIFT` + `S`, or `/`).

Will only work on pages with the top search bar (not the `/search/` page)

### CODE REVIEW

**Comment by kodster28** [comment]
FYI for @Cherry and @irvinebroque who independently brought this up to me. Didn't know we actually already had this functionality is our docs... but seems best to match the growing standard for CTRL + K.

File: assets/search.redirect.ts [hunk 1]
```diff
@@ -21,8 +21,8 @@
       return location.assign(redirect);
     }
 
-    // is '/' or SHIFT+'s'
-    if (key === 191 || (ev.shiftKey && key === 83)) {
+    // is '/' or SHIFT+'s' or CTRL+'k'
+    if (key === 191 || (ev.shiftKey && key === 83) || (ev.ctrlKey && key === 75)) {
```
Comment (irvinebroque) [inline assets/search.redirect.ts]: Think what you want here is command-k, not ctrl-k (see Stripe and other docs for reference)

**Comment by irvinebroque** [comment]
🎉  nice one!

File: assets/search.redirect.ts (hunk 1 above)
Comment (kodster28) [reply assets/search.redirect.ts]: So close, lol

**Review by dcpena — APPROVED** [review]

**Review by Cherry — COMMENTED** [review]
Oh cool! `metaKey` on Windows is the Windows key though unfortunately, and won't be captured by browsers. Using `Ctrl` for Windows would be ideal.

Algolia's approach [seems to be](https://github.com/algolia/docsearch/blob/2c64362067376a76575b7b5131972f71c920a406/packages/docsearch-react/src/useDocSearchKeyboardEvents.ts#L42):

```js
 (event.key === 'k' && (event.metaKey || event.ctrlKey))
```

**Comment by WalshyDev** [comment]
Yeah, cmd is best for Mac but ctrl for Windows/Linux. Don't you love cross-platform 😅

**Comment by kodster28** [comment]
Actually, @dcpena, could you review the new styling as well?

Sorta stole from Stripe's docs and added one of our shortcuts to be visible in the top search bar (on doc pages, not the main/tutorial pages).

Should also work in mobile, btw.

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**